egs/yomdle_russian/v1/local/augment_data.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# Copyright   2018 Hossein Hadian
+#             2018 Ashish Arora
+
+# Apache 2.0
+# This script performs data augmentation.
+
+nj=4
+cmd=run.pl
+feat_dim=40
+verticle_shift=0
+echo "$0 $@"
+
+. ./cmd.sh
+. ./path.sh
+. ./utils/parse_options.sh || exit 1;
+
+srcdir=$1
+outdir=$2
+datadir=$3
+
+mkdir -p $datadir/augmentations
+echo "copying $srcdir to $datadir/augmentations/aug1, allowed length, creating feats.scp"
```

Comment:
echo statements should usually start with "$0: " so that it's clear which script produced them.

Reply:
sorry, local/augment_data.sh is not used yet for yomdle_russian setup. I will remove it.

Reply:
oh, ok, I got it now... making changes.